integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -243,6 +244,42 @@ public void verifyAdminServerExternalService(String username, String password) t
     }
   }
 
+  /**
+   * Verify that we have the channel set for the cluster
+   *
+   * @param protocol
+   * @param port
+   * @throws Exception
+   */
+  public void verifyHasClusterServiceChannelPort(String protocol, int port) throws Exception {
+    if (!TestUtils.checkHasServiceChannelPort(
+        this.getDomainUid() + "-cluster-" + this.clusterName, protocol, port)) {
+      throw new RuntimeException(
+          "FAILURE: Cannot find channel port in cluster, but expecting one"
+              + port
+              + "/"
+              + protocol);
+    }
+  }
+
+  /**
+   * Verify that we have do not have the channel set for the cluster
+   *
+   * @param protocol
+   * @param port
+   * @throws Exception
+   */
+  public void verifyNotClusterServiceChannelPort(String protocol, int port) throws Exception {
```

Comment:
This method doesn't look like it's getting called.

Reply:
Yep, I'll remove it.